src/renderer/index.js

```
@@ -14,11 +37,12 @@ import { initAppManager } from "./actions/apps";
 import "./types";
 import { FsManagerBuilderRPCService } from "./actions/local-sync/fs-manager-builder.rpc-service";
 import { clearStoredLogs } from "./lib/proxy-interface/loggerService";
+import logger from "utils/logger";
```

Biome (2.3.14)

[error] Shouldn't redeclare 'logger'. Consider to delete it or rename it.

'logger' is defined here:


(lint/suspicious/noRedeclare)